### PR TITLE
Authhelper: improve username field choice

### PR DESCRIPTION
**Pull request by psiinon**

### CODE REVIEW

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

File: addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/AuthUtils.java [hunk 1]
```diff
@@ -81,6 +81,7 @@ public class AuthUtils {
 
     public static final String[] HEADERS = {HttpHeader.AUTHORIZATION};
     public static final String[] JSON_IDS = {"accesstoken", "token"};
+    private static final String[] USERNAME_FIELD_INDICATORS = {"email", "user"};
```
Comment (kingthorin) [inline addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/AuthUtils.java:84]: Could I talk you into adding:
- `EmailAddress`
- `uname`
- `username`

Based on having gone through the VWAD "online" list and inspected a bunch of "login" forms.

Comment (psiinon) [reply addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/AuthUtils.java:84]: We do a "contains" check on the lowercase values, so `EmailAddress` and `username` will already work 😁 
V happy to add others - `name` maybe?

Comment (kingthorin) [reply addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/AuthUtils.java:84]: "name" might be problematic as it'll match firstname, lastname in registration forms.

Comment (psiinon) [reply addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/AuthUtils.java:84]: Do sites have registration forms on the same page as login ones? I guess some might...

**Comment by kingthorin** [comment]
I also just went through some online businesses and encountered `signInName` twice.

File: addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/AuthUtils.java (hunk 1 above)
Comment (kingthorin) [reply addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/AuthUtils.java:84]: I also just went through some online businesses and encountered `signInName` twice.

**Review by kingthorin — APPROVED** [review]
Thanks!

**Pull request merged**